src/Microsoft.HttpRepl/Preferences/IPreferencesProvider.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace Microsoft.HttpRepl.Preferences
+{
+    public interface IPreferencesProvider
+    {
+        string PreferencesFilePath { get; }
```

Comment:
Does this have to be exposed?  Seems like it's revealing the backing store, which should be an implementation detail (e.g. I should be able to swap in a non-file based implementation).

Reply:
Nope, doesn't need to be exposed. Removed.